### PR TITLE
83323 update application

### PR DESCRIPTION
**Pull request by huynt-3640**
## Related Tickets
- [#83323](https://edu-redmine.sun-asterisk.vn/issues/83323)

## WHAT (optional)
- Rple doanh nghiệp cập nhật thông tin 1 đơn apply của 1 ứng viên

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image
Form Add Interview:
image
image
Update Interview:
image
Show:
image

### CODE REVIEW

**Comment by huynt-3640** [comment]
ready

File: app/views/enterprise/applications/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,114 @@
...
+
+<div class="modal fade" id="newInterviewModal" tabindex="-1" aria-labelledby="newInterviewModalLabel" aria-hidden="true">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="newInterviewModalLabel"><%= t "enterprise.applications.add_interview" %></h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <%= form_with model: @interview_process, url: enterprise_application_interview_processes_path(application_id: @application.id), method: :post, local: true do |f| %>
```
Comment (ghost) [inline app/views/enterprise/applications/show.html.erb]: dùng partial
bị lặp code với bên trên

**Review by ghost — COMMENTED** [review]
cmt

**Comment by huynt-3640** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**